envars.example.json

```diff
@@ -0,0 +1,14 @@
+{
+    "NEMSSubscriptionCreateFunction": {
+        "PDS_JWT_SUB": "PDS API Key",
+        "PDS_JWT_ISS": "PDS API Key",
+        "PDS_API_KEY": "PDS API Key",
+        "PDS_JWKS_KID" : "int-2",
+        "PDS_JWT_RSA_PRIVATE_KEY": "-----BEGIN RSA PRIVATE KEY----- PDS PRIVATE KEY -----END RSA PRIVATE KEY-----",
+        "PDS_JWT_AUD" : "https://int.api.service.nhs.uk/oauth2/token",
+        "PDS_BASE_URL" : "https://int.api.service.nhs.uk"
+    },
+    "EmailCareProviderFunction": {
+        "NOTIFY_API_KEY" : "hansstaging-6cfde5cc-72e2-4e66-8bea-c87659099f74-a5c3df98-7760-497d-90d2-306e52561ec4"
```

Comment:
Let's crop that out - now it's the only one there it stands out like a sore thumb, and  anyone using this will want their own anyway.